src/com/oltpbenchmark/benchmarks/auctionmark/AuctionMarkWorker.java

```diff
@@ -295,9 +303,9 @@ public AuctionMarkWorker(int id, AuctionMarkBenchmark benchmark) {
         this.profile = new AuctionMarkProfile(benchmark, benchmark.getRandomGenerator());
         
         boolean needCloseAuctions = (AuctionMarkConstants.CLOSE_AUCTIONS_ENABLE); //&& id == 0);
-        this.closeAuctions_flag.set(needCloseAuctions);
+        this.closeAuctions_flag.set(false);
```

Comment:
Don't start everyone with the need to close auctions. Only one person wins this, and then no one else has any mappings.